pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "quranref"
+version = "0.1.1"
+description = ""
+authors = ["Kashif Iftikhar <kashif@compulife.com.pk>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "3.10.6"
+urllib3 = "1.26.15"
+requests = "2.28.1"
+fastapi = "0.95.2"
+typer = "0.9.0"
+arango-orm = "0.7.0"
+python-dotenv = "1.0.0"
+pydantic = "1.10.8"
+
+
+[tool.poetry.group.dev.dependencies]
+ipython = "8.13.2"
+uvicorn = "0.22.0"
+ruff = "^0.0.270"
+black = "^23.3.0"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.scripts]
+quranref = 'quranref.cli:main'
+quranref_populate = 'quranref.scripts.populate:main'
```

Comment:
Why have you uncommented this line? I think you still don't understand how to use typer :-( `quranref` is going to be the main command and other commands are going to be subcommands. Like `quranref populate`